modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/types.ts

```diff
@@ -119,6 +119,18 @@ export type TAction =
 			};
 			type: TYPES.SET_SELECTED_OBJECT_DEFINITION_NODE;
 	  }
+	| {
+			payload: {
+				newPosition: {
```

Comment:
@igor-franca, I think its better we rename this prop to `newObjectDefinitionNodePosition`, what do you think?